packages/gallery/src/components/gallery/proGallery/galleryContainer.js

```diff
@@ -84,6 +84,8 @@ export class GalleryContainer extends React.Component {
     this.handleNewGalleryStructure();
     this.eventsListener(GALLERY_CONSTS.events.APP_LOADED, {});
     this.videoScrollHelper.initializePlayState();
+    const currentItem = this.galleryStructure.items[this.props.activeIndex || 0];
+    this.videoScrollHelper.initScroll(currentItem.offset);
```

Comment:
why do you need all this new code. you have a function for this called onGalleryScroll

Reply:
it just didn't feel right calling the onGalleryScroll (which also fires an event - GALLERY_SCROLLED) since initialize the left/top with the right values